esque/cli/commands.py

```diff
@@ -311,10 +311,21 @@ def transfer(
             return
 
         state.config.context_switch(to_context)
-        _produce_from_file(topic, to_context, working_dir, avro)
+        _produce_from_files(topic, to_context, working_dir, avro)
 
 
-def _produce_from_file(topic: str, to_context: str, working_dir: pathlib.Path, avro: bool):
+@esque.command("produce", help="Produce messages from directory based on output from transfer command")
+@click.argument("topic", required=True)
+@click.option(
```

Comment:
Now that I think about it: In my opinion it is unreasonable to have required options. We should probable either make it an argument or set a default. E.g. fall back to the current directory if nothing is given. What do you think @swenzel ?

Reply:
I also thought about fallback to current directory.

Reply:
Well `kapp` also has required options... They're a little awkward since you don't expect them and it's not very obvious from the command description that they're required.  
Therefore I'd rather do it like 'docker build': Use an argument which requires a path. Even when it's `.`.

Also please don't forget to describe the arguments in the command help parameter. That works a little different from describing options. Some stupid example:
```python
@click.command(help=f"""
This command does <action> on <directory>
Where <action> may be one of:
 - dothis: for doing this
 - dothat: for doing that

And <directory> is a directory full of stuff.
""")
@click.argument("action", metavar="<action>")
@click.argument("directory", metavar="<directory>", type=pathlib.Path)
def do(action: str, directory: pathlib.Path):
  pass
```